packages/web-client/app/components/card-pay/layer-one-connect-modal/index.hbs

```diff
@@ -1,10 +1,17 @@
 <Boxel::Modal
+  @isOpen={{@isOpen}}
   @onClose={{@onClose}}
-  style={{css-var boxel-modal-max-width="45rem"}}
+  style={{css-var boxel-modal-max-width="45rem" boxel-modal-offset-top="0px"}}
   class="layer-one-connect-modal"
   data-test-layer-one-connect-modal
 >
-  <Boxel::CardContainer class="layer-one-connect-modal__card">
-    Connect...
-  </Boxel::CardContainer>
+  <div class="layer-one-connect-modal__scroll-wrapper">
+    <Boxel::CardContainer
+      class="layer-one-connect-modal__card"
+      tabindex="0"
+      {{did-insert this.focus}}
+    >
+      Connect...
+    </Boxel::CardContainer>
+  </div>
```

Comment:
Would we prefer to put a scroll wrapper around a CardContainer, or a way to make a CardContainer scrollable?